examples/uefi/libvirt.tf

```diff
@@ -0,0 +1,43 @@
+provider "libvirt" {
+    uri = "qemu:///system"
+}
+
+resource "libvirt_network" "dev-cluster" {
+   name = "dev-cluster"
+   domain = "dev-cluster.local"
+   mode = "nat"
+   addresses = ["10.0.200.0/24"]
+}
+
+resource "libvirt_volume" "debian9-base" {
+  name = "debian9-base"
+  source = "http://localhost:8000/debian9-uefi.qcow2"
+}
+
+resource "libvirt_volume" "volume" {
+  name = "vm${count.index}"
+  base_volume_id = "${libvirt_volume.debian9-base.id}"
+  count = 2
+}
+
+resource "libvirt_domain" "domain" {
+  name = "vm${count.index}"
+  memory = "256"
+  vcpu = 1
+  nvram = "/var/lib/libvirt/qemu/nvram/vm${count.index}_VARS.fd"
```

Comment:
So, this one is a binary file and is actually the destination (backing storage) of the UEFI memory after the domain is deployed. 

I am creating it manually on the tests because the libvirt version installed on the Travis CI build image is not doing it. On the libvirt version I'm using on my test systems (Debian 9 with a backported libvirt 3.0.0) this file is created automatically from the template.